user-guide/Advanced_Functionality/Databases/Elasticsearch_database/Installing_Elasticsearch_on_Linux.md

```diff
@@ -0,0 +1,116 @@
+---
+uid: Installing_Elasticsearch_on_Linux
+---
+
+# Installing Elasticsearch on a Linux machine
+
+If you want to use an Elasticsearch cluster for your DMS (which is required to use the [Cassandra Cluster](xref:Migrating_the_general_database_to_a_DMS_Cassandra_cluster) feature), install Elasticsearch on a Linux machine as detailed below.
+
+1. Install the Elasticsearch software on  the Linux machine as described under [Installing from the RPM repository](https://www.elastic.co/guide/en/elasticsearch/reference/6.8/rpm.html#rpm-repo) in the official Elasticsearch documentation.
+
+1. Make sure the firewall ports are open for Elasticsearch. Elasticsearch operates on TCP port 9200 and TCP port 9300.
+
+   - There is a default firewall on Linux, but this is disabled by default. To enable the firewall, use the following command:
+
+     `$ systemctl start firewalld.service`
+
+   - To add the correct ports to the firewall, you can for example use the following commands:
+
+     - `$ firewall-cmd --add-port=9200/tcp --permanent`
+
+     - `$ firewall-cmd --add-port=9300/tcp --permanent`
+
+     - `$ firewall-cmd --reload`
+
+   > [!IMPORTANT]
+   >
+   > - These commands could be **different depending on the Linux distribution** you are using. Refer to [the official Elastic page on installing Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/install-elasticsearch.html) for detailed information.
+   > - If you connect to your Linux server with **SSH**, you must **immediately exclude port 22**, or you will be locked out of the session. Use the following command for this: `$ firewall-cmd --zone=public --add-port=22/tcp`
+
+1. Mount the data folder to the data disk.
+
+   > [!NOTE]
+   >
+   > - The folder where the Elasticsearch data is stored is configured in *elasticsearch.yaml*, in the property *path.data*.
+   > - To verify on which disk the data is mounted, execute the "df" command.
+
+1. Configure the user rights for the Elasticsearch user/group in the data folder.
+
+   You can do this with the following command:
+
+   `$ chown -R elasticsearch:elasticsearch /directoryname_from_path.data`
+
+1. Configure the *elasticsearch.yml* files.
+
+   - Make sure the Elasticsearch service is stopped. To stop the Elasticsearch service, use the following command:
+
+     `$ sudo systemctl stop elasticsearch.service`
+
+   - To change *elasticsearch.yml* files, you can use the following command:
+
+     `$ vi /etc/elasticsearch/elasticsearch.yml`
```

Comment:
When using vi, might as well link to a vi reference for dummies: e.g. https://www.cs.colostate.edu/helpdocs/vi.html 
Can also recommend nano as it is more simple to work with. Vi is not a very intuitive editor.